eternalcore-core/src/main/java/com/eternalcode/core/feature/ignore/UnIgnoreCommand.java

```diff
@@ -1,28 +1,35 @@
 package com.eternalcode.core.feature.ignore;
 
 import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.event.EventCaller;
 import com.eternalcode.core.injector.annotations.Inject;
 import com.eternalcode.core.notice.NoticeService;
 import com.eternalcode.core.user.User;
 import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
 import dev.rollczi.litecommands.annotations.context.Context;
 import dev.rollczi.litecommands.annotations.execute.Execute;
 import dev.rollczi.litecommands.annotations.permission.Permission;
-import dev.rollczi.litecommands.annotations.command.Command;
+import org.bukkit.Server;
 
 import java.util.UUID;
 
+
 @Command(name = "unignore")
 @Permission("eternalcore.ignore")
 class UnIgnoreCommand {
 
-    private final IgnoreRepository repository;
+    private final IgnoreService ignoreService;
     private final NoticeService noticeService;
+    private final EventCaller eventCaller;
+    private final Server server;
 
     @Inject
-    public UnIgnoreCommand(IgnoreRepository ignoreRepository, NoticeService noticeService) {
-        this.repository = ignoreRepository;
+    public UnIgnoreCommand(IgnoreService ignoreService, NoticeService noticeService, EventCaller eventCaller, Server server) {
+        this.ignoreService = ignoreService;
         this.noticeService = noticeService;
+        this.eventCaller = eventCaller;
+        this.server = server;
     }
 
```

Comment:
```java
    private final IgnoreService ignoreService;
    private final NoticeService noticeService;

    @Inject
    public UnIgnoreCommand(IgnoreService ignoreService, NoticeService noticeService) {
        this.ignoreService = ignoreService;
        this.noticeService = noticeService;
    }
```

Reply:
In the IngoreCommand/UnIngoreCommand class, the EventCaller and Server are unnecessary.